src/scss/ucf-spotlight.scss

```diff
@@ -0,0 +1,180 @@
+$screen-lg-min: 1200px;
+
+// Common to all three layouts
+.spotlight-horizontal,
+.spotlight-vertical,
+.spotlight-square {
+
+  .spotlight-header,
+  .spotlight-copy,
+  .spotlight-link {
+    font-family: $font-family-sans-serif;
+  }
+
+  .spotlight-copy {
+
+    .spotlight-link {
+      color: #337ab7;
+
+      &:hover {
+        text-decoration: underline;
+      }
+    }
+  }
+
+  .btn {
+    @include button-size($btn-padding-y, $btn-padding-x, $font-size-base, $btn-border-radius);
+    @include transition($btn-transition);
+
+    border: $input-btn-border-width solid transparent;
+    border-width: $btn-border-width;
+    display: inline-block;
+    font-size: $btn-font-size;
+    font-weight: $btn-font-weight;
+    letter-spacing: $letter-spacing-base;
+    line-height: $btn-line-height;
+    text-align: center;
+    text-transform: uppercase;
+    user-select: none;
+    vertical-align: middle;
+    white-space: nowrap;
+
+    @include media-breakpoint-up(md) {
+      font-size: $btn-font-size-md;
+    }
+
+    // Share hover and focus styles
+    @include hover-focus {
+      text-decoration: none;
+    }
+
+    &:hover,
+    &.active,
+    &:active,
+    &.focus,
+    &:focus {
+      border-color: transparent;
+    }
+
+    &:focus,
+    &.focus {
+      box-shadow: $btn-focus-box-shadow;
+      color: initial;
+      outline: 0;
+    }
+
+    // Disabled comes first so active can properly restyle
+    &.disabled,
+    &:disabled {
+      @include box-shadow(none);
+      cursor: $cursor-disabled;
+      opacity: .65;
+    }
+
+    &:active,
+    &.active {
+      @include box-shadow($btn-focus-box-shadow, $btn-active-box-shadow);
+      background-image: none;
+    }
+  }
+
+  .btn-primary {
+    @include button-variant($btn-primary-color, $btn-primary-bg, $btn-primary-border);
+  }
+
+}
+
+// Horizontal spotlight layout
+.spotlight-horizontal {
+  padding: 30px;
+
+  .spotlight-header {
+    color: #fff;
+    font-family: $font-family-condensed;
+    font-size: 50px;
+    font-weight: 400;
+    letter-spacing: .2em;
+    line-height: normal;
+    margin-bottom: 25px;
+    margin-top: 21px;
+    text-transform: uppercase;
+
+    @media (min-width: $screen-md-min) {
+      font-size: 60px;
+    }
+
+    @media (min-width: $screen-lg-min) {
+      font-size: 70px;
+    }
+  }
+
+  .spotlight-copy {
+    color: #fff;
+    margin-bottom: 24px;
+  }
+}
+
+// Square spotlight layout
+.spotlight-square {
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  min-height: 293px;
+  padding: 40px;
+  position: relative;
+
+  .spotlight-header,
+  .spotlight-copy {
+    color: #fff;
+  }
+
+  .spotlight-header,
+  .spotlight-copy,
+  .spotlight-link {
+    text-align: center;
+  }
+
+  .spotlight-header {
+    font-size: 26px;
+    font-weight: 900;
+    margin-bottom: 15px;
+    text-shadow: 1px 1px 1px #000;
+  }
+
+  .spotlight-copy {
+    line-height: 24px;
+  }
+
+  .btn {
+    width: 213px;
```

Comment:
Remove this fixed width (see notes on .spotlight-btn-wrapper margin-left).  If you want to make sure the button spans a minimum amount of space, use min-width instead.